addons/pos_restaurant/static/src/overrides/components/navbar/back_button/back_button.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="pos_restaurant.BackButton" t-inherit="point_of_sale.BackButton" t-inherit-mode="extension">
+         <xpath expr="//span//span" position="replace">
```

Comment:
give a class or a name to the span you want to target and xpath by class or name otherwise someone can add something without thinking about this xpath